planning/planning_debug_tools/Readme.md

```diff
@@ -54,6 +54,12 @@ PlotVelocityOverArclength('v_motion_avoid', motion_avoid, tracker_time)
 PlotVelocityOverArclength('v_motion_smoother_latacc', motion_smoother_latacc, tracker_time)
 PlotVelocityOverArclength('v_motion_smoother', motion_smoother, tracker_time)
 
+PlotAccelerationOverArclength('v_behavior_path', behavior_path, tracker_time)
```

Comment:
@takayuki5168 
v_behavior_path should be a_behavior_path please fix other names too 
```suggestion
PlotAccelerationOverArclength('a_behavior_path', behavior_path, tracker_time)
```

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1782/commits/52a8962256e51f63da7e529cad5691f09a61b912
DONE